flink/flink-scala-2.12/pom.xml

```diff
@@ -254,6 +254,17 @@
       <groupId>com.mashape.unirest</groupId>
       <artifactId>unirest-java</artifactId>
```

Comment:
I think it makes more sense to raise this dependency and not play around with the transitive dependency.
From version 3.0 this library uses  `com.google.code.gson » gson` and no longer `org.json » json`.
We also use GSON in many other places in Zeppelin.

Reply:
I am not a Zeppelin committer. I never use Zeppellin. I'm just trying to help out but have very limited time to be experimenting.